CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## v1.19.0 (unreleased)
```

Comment:
Add a comment about the new import path (breaking change) and remove "unreleased". Once merged into master you can tag the repo with v1.19.0.